mpf/devices/segment_display/segment_display.py

```diff
@@ -20,6 +20,29 @@
     from mpf.core.platform import SegmentDisplayPlatform    # pylint: disable-msg=cyclic-import,unused-import; # noqa
 
 
+class SegmentDisplayState:
+
+    """Current State."""
+
+    def __init__(self, text: str, colors: List[RGBColor],
+                 flashing: FlashingType, flash_mask: Optional[str] = None):
+        """Class initializer."""
+        self.text = text
+        self.colors = colors
+        self.flashing = flashing
+        self.flash_mask = flash_mask
+
+    def __repr__(self):
+        """Return str representation."""
+        return '<TextStackEntry: {} (colors: {}, flashing: {} flashing_mask: {}) >'.format(
```

Comment:
Should we change 'TextStackEntry' to 'SegmentDisplayState'?

Reply:
Guess the stack contains slightly more. But the idea sounds good